src/utility/control_sil/launch/control_sil_test.launch

```diff
@@ -0,0 +1,16 @@
+<!-- Launch file for the control software in the loop testing node -->
+
+
+<launch>
+  <!-- type-> executable , pkg->package, name-> node name-->
+  <node type="control_sil_sim" pkg="control_sil" name="control_sil_sim" output="screen" >
```

Comment:
Are these parameters for black car? If so, maybe we should note somewhere in the wiki or top-level README that all physical and hardcoded parameters are assumed to be related to the black car, by default

Reply:
For software in the loop testing, the wheelbase doesn't quite matter because the control will work regardless of the wheelbase.